scripts/gha/build_desktop.py

```diff
@@ -55,7 +55,17 @@ def append_line_to_file(path, line):
     with open(path, "a") as file:
       file.write("\n" + line + "\n")
 
-def install_cpp_dependencies_with_vcpkg(arch):
+def install_x86_support_libraries():
+  """Install support libraries needed to build x86 on x86_64 hosts."""
+  if utils.is_linux_os():
+    utils.run_command(['apt', 'install', 'gcc-multilib', 'g++-multilib'], as_root=True)
+    utils.run_command(['dpkg', '--add-architecture', 'i386'], as_root=True)
+    utils.run_command(['apt', 'update'], as_root=True)
```

Comment:
Do you need to pass an argument to "apt" to force it to say "yes" to all prompts?

Reply:
Fortunately, I didnt have to pass any on my machine or on github machines. I am hesitant to add an automatic "yes" as this installs system level packages and it would be more transparent for our users to be prompted for confirmation. But I will definitely keep an eye out for any issues caused by not doing this automatically.